frontend/components/form/EditorField.vue

```diff
@@ -1,42 +1,40 @@
 <script setup lang="ts">
 import { useSlots } from 'vue'
-import { type EditorField, isValid } from '@/lib/editor'
+import { type EditorField, type EditorValue, isValid } from '@/lib/editor'
+
+const { t } = useI18n()
+const tt = (key: string) => t(`components/form/EditorField.${key}`)
 
 interface Props {
   editorField: EditorField<any, keyof any>
-  helpText?: string
-  startHelpTextExpanded?: boolean
+  editorValue: EditorValue<any, keyof any>
```

Comment:
I'm no TypeScript whiz, but I'd imagine `keyof any` would just evaluate to `any`. If there's supposed to be some relationship between these two generic parameters, I'd capture it with something like `<T, keyof T>`.

Reply:
Great call - done.